README.md

```diff
@@ -65,3 +65,8 @@ BotBlock.hashText = True
 BotBlock.nameIsTextHash = True
 BotBlock.generate('/tmp/')
 ```
+
+### Example Images
+![Sample Captcha #1](/examples/sample1.png)
```

Comment:
```suggestion
![Sample Captcha #1](/examples/sample1.png "First Sample Image")
```

Add default text for all of the images, in case they don't load